string filepath
string frame
---
sensor_msgs/PointCloud2 cloud